src/frame/js/utils/dom.js

```diff
@@ -1,9 +1,12 @@
 import isMobile from 'ismobilejs';
 
+// Rewrite the loaders to avoid the style-loader used in the host part
+import hostStyles from '!css-loader?modules!less-loader!../../../host/stylesheets/iframe.less';
```

Comment:
so great thing about using css modules is that webpack will generate a classname that won't clash with anything (`_2TELtk5nDKlQudVSivRjpt`). It's really perfect for styles in the host page.

You can then import the stylesheet and find the classname like this `hostStyles.locals.widgetOpened`